meta/Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "_README.md"
 
 [dependencies]
 maplit = "^0.1.5"
-pest = { path = "../pest", version = "^1.0" }
+pest = "^1.0"
```

Comment:
Why not have the path dependency here? It makes development much easier.

Reply:
This has to be changed because `pest_meta` depends on both `pest_derive` and `pest` and with the significant changes made to `pest`, `pest_meta` won't compile. Also, in general, both `pest` and `pest_derive` should point to the most recently released versions of those libraries since this is the library that `pest_derive` depends on.